server.go

```diff
@@ -1611,17 +1611,17 @@ func (s *server) Start() error {
 		}
 		cleanup = cleanup.add(s.utxoNursery.Stop)
 
-		if err := s.chainArb.Start(); err != nil {
+		if err := s.breachArbiter.Start(); err != nil {
```

Comment:
Heh, looks like the very same bug fix is actually implemented in this _very_ old PR: https://github.com/lightningnetwork/lnd/pull/1783 

I wonder if we should cherry-pick it as is, as it attempts to address other known dependency startup issues like what you set out to fix here.

Reply:
Turns out that pr is pretty outdated and some of the suggested changes were already made so I just added the needed changes here manually.